src/main/java/mate/controller/DisplayAllDriversController.java

```diff
@@ -0,0 +1,27 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers")
```

Comment:
Consider giving the servlet a more descriptive urlPattern, such as '/drivers/all', to clearly indicate that this servlet is responsible for displaying all drivers. 🚗

Reply:
/drivers - (get all drivers)

Reply:
The comment from check list

Reply:
Maybe better chnge the check list because a bit leads off from right way